guides/intergration/oidc.md

```diff
@@ -0,0 +1,120 @@
+# OpenID Connect
+
+Trento integrates with identity providers that use the OpenID Connect **(OIDC)** protocol to authenticate users accessing the console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external identity provider **(IDP)**.
+
+## Enabling OIDC
+
+The OIDC authentication is **disabled by default**.
+
+Provide the following environment variable to enable OIDC feature when starting Trento.
+
+```
+ENABLE_OIDC=true
+OIDC_CLIENT_ID=<<OIDC_CLIENT_ID>>
+OIDC_CLIENT_SECRET=<<OIDC_CLIENT_SECRET>>
+OIDC_BASE_URL=<<OIDC_BASE_URL>>
+OIDC_CALLBACK_URL=<<OIDC_CALLBACK_URL>>
+```
+
+## Enabling OIDC in Development
+
+Enable OIDC in the development environment using Docker and [Keycloak](https://github.com/keycloak/keycloak) as a simulated IDP.
+
+### Starting the Keycloak Identity Provider
+
+Use a custom Docker profile to start Keycloak as IDP for local development.
+
+Start the Docker containers with the `idp` profile:
+
+```
+docker compose --profile idp up
+```
+
+Keycloak server can be accessed at [http://localhost:8081](http://localhost:8081)
+
+### Create OIDC configuration
+
+1. Create a new runtime configuration `dev.local.exs` in `config` directory.
+
+1. Enable OIDC in runtime config:
```

Comment:
Why `1.` 3 times in a row?
I know that markdown will render properly, but it looks more difficult to edit in the future

Reply:
The documentation team uses the `1. `notation for consistency and to make editing easier. 
It is easy to forget to adjust the numbering when you move stuff around. 
For me personally it is even easier to edit like this because i must not think about the numbering haha. 

We can remove it if you wan't as not everybody is used it, so i see that it can make stuff harder.
What do you think ?

Reply:
Up to you, no hard feelings hehe